doc/user/content/overview/materialize-cdc.md

```diff
@@ -0,0 +1,237 @@
+---
+title: "Materialize CDC"
+description: "The Materialize CDC format is a format for change datafeeds that has been designed especially to prevent record duplication or mistaken missequencing."
```

Comment:
I might say instead ".. designed especially to be resilient to record duplication and reordering." It doesn't prevent these from happening, but it does prevent us from producing the wrong answers when they do happen.

Reply:
I made this "The Materialize CDC format is a format for change datafeeds that has been designed especially to prevent errors resulting from record duplication or missequencing."